### ISSUE TITLE
Opencvjs 4.0 WASM

### ISSUE THREAD

**Issue by Neuroforge** [haoking/opencvjs#3]
Hello,

Thank you for doing this!

This repo has many functions that i am missing from official OpenCVjs.

I also need to add cv.round().

Are the bindings available to rebuild/regenerate? I would like to rebuild OpenCVjs using WASM and Emscripten to make it faster.

Regards,

**Comment by haoking** [comment]
RIght, WASM and Emscripten do make it faster. And I finally have time to work on the project again. I will try to add some more functions and rebuild them in the next version. Thanks for your help.

**Comment by Neuroforge** [comment]
Here they have used threads, wasm and smid. ;)

https://github.com/opencv/opencv/pull/15371

I think you should push your changes and fixes to OpenCV's main repo. Would be a wonderful help.
